sycl/doc/GetStartedGuide.md

```diff
@@ -663,6 +699,12 @@ which contains all the symbols required.
 * The NVIDIA OpenCL headers conflict with the OpenCL headers required for this
   project and may cause compilation issues on some platforms
 
+### ROCm back-end limitations
+
+* Backend is only supported on Linux
+* The only combination tested is Ubuntu 18.04 with ROCm 4.1 using a Vega20 gfx906
+* Judging from the current [test](https://github.com/zjin-lcf/oneAPI-DirectProgramming) results, there is still a lot of room for improvement in ROCm back-end support. The current problems include three aspects. The first one is at compile time: the `barrier` and `atomic` keywords are not supported. The second is at runtime: when calling `hipMemcpyDtoHAsync` ROCm API, the program will cause an exception if the input data size is too large. The third is calculation accuracy: the ROCm backend has obvious errors in the calculation results of some float type operators
```

Comment:
Please, try to keep lines 80 chars long. It's ok to have new lines in the list items, Markdown can understand that.

Also, out of curiosity. Originally SYCL inherited its accuracy limits from OpenCL. CUDA has much stricter limitations (see https://intel.github.io/llvm-docs/cuda/cuda-vs-opencl-math-builtin-precisions.html). Do you observe a similar effect?